waterArduino.INO

```diff
@@ -3,12 +3,20 @@ Servo waterServo;  //creates the name of your servo
 int sensPin = 0;  //assignes a pin for the soil sensor
```

Comment:
Hi Tera!

On the left is the student's code as it was in the email.  On the right is my version of the code.  Lines on the left in pink means they were changed in my version.  Lines on the right in green show what changed.